app/views/content_pages/interruption_page.html.slim

```diff
@@ -4,11 +4,8 @@
   .interrupt-panel.govuk-grid-column-full
     h1 What to expect during the training
     .gem-c-govspeak
-      - if @model.formative?
-        = translate_markdown t('interruption.formative')
-      - else
-        = translate_markdown t('interruption.summative')
-
+      = translate_markdown t('interruption.formative') if @model.formative?
+      = translate_markdown t('interruption.summative') if @model.summative?
```

Comment:
Laura says she does think we need the `formative?` here - can you check with her? @peterdavidhamilton @agoldstone93 @mo-zag